DeploymentCloud/Deployment.Common/CosmosDB/configgenConfigs.json

```diff
@@ -41,6 +41,8 @@
     "interactiveQueryDefaultContainer" : "defaultdx",
     "sparkClusterName" : "$sparkName",
     "subscriptionId" : "keyvault://$servicesKVName/$clientSecretPrefix-subscriptionId",
-    "sparkConnectionString" : "keyvault://$servicesKVName/$serviceSecretPrefix-livyconnectionstring-$sparkName"
+    "sparkConnectionString" : "keyvault://$servicesKVName/$serviceSecretPrefix-livyconnectionstring-$sparkName",
+    "sparkRegion" : "$resourceLocation",
```

Comment:
Probably it would be better to introduce a new param for the spark location (resourceLocationForSpark) in common.parameters.txt?

Reply:
databricks location should be the same as rest of resources